client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -53,6 +53,7 @@ import InfoPopover from 'calypso/components/info-popover';
 import ExternalLink from 'calypso/components/external-link';
 import HeaderCart from 'calypso/my-sites/checkout/cart/header-cart';
 import AddDomainButton from 'calypso/my-sites/domains/domain-management/list/add-domain-button';
+import { logmeinUrl } from 'calypso/lib/logmein';
```

Comment:
modifying this file probably no longer required - links are handled automagically